pallets/registrar/src/lib.rs

```diff
@@ -77,7 +77,8 @@ pub mod pallet {
     #[derive(DefaultNoBound)]
     pub struct GenesisConfig<T: Config> {
         /// Para ids
-        pub para_ids: Vec<(ParaId, ContainerChainGenesisData<T::MaxLengthTokenSymbol>)>,
+        pub para_ids: Vec<(ParaId, ContainerChainGenesisData)>,
```

Comment:
You can remove `T::MaxLengthTokenSymbol` from config now, right?

So basically instead of having a configurable max length, we now fix it to 255 in the struct definition. Not ideal but I guess it will be good enough for our use case. There's an old TODO about adding bounds to the other vecs, if we follow this approach it will be easier to do, the alternative was to add like 5 generics or a custom trait to each struct, which looks quite ugly.